### PR TITLE
fixed bug of storage client for index scan

### PR DESCRIPTION
**Pull request by bright-starry-sky**

### CODE REVIEW

**Review by panda-sheep — APPROVED** [review]
Good，LGTM

File: src/common/clients/storage/GraphStorageClient.cpp [hunk 1]
```diff
@@ -479,7 +479,7 @@ GraphStorageClient::lookupIndex(GraphSpaceID space,
         req.set_return_columns(returnCols);
 
         cpp2::IndexSpec spec;
-        spec.set_contexts(std::move(contexts));
```
Comment (Shylock-Hg) [inline src/common/clients/storage/GraphStorageClient.cpp:482]: what's the bug?

Comment (darionyaphet) [reply src/common/clients/storage/GraphStorageClient.cpp:482]: const should not move ？

Comment (Shylock-Hg) [reply src/common/clients/storage/GraphStorageClient.cpp:482]: But the origin code is not const.

**Review by critical27 — APPROVED** [review]

**Pull request merged**